doc/ale.txt

```diff
@@ -564,7 +568,6 @@ g:ale_completion_delay                                 *g:ale_completion_delay*
 
   See |ale-completion|
 
-
```

Comment:
Leave the double spacing in place. I tend to write two blank lines between each option.